pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -793,7 +828,8 @@ def test_with_offset(self):
         assert self.d + BusinessHour(n=3) == expected
 
     def test_eq(self):
-        for offset in [self.offset1, self.offset2, self.offset3, self.offset4]:
+        for offset in [self.offset1, self.offset2, self.offset3, self.offset4,
```

Comment:
Can you use `pytest.mark.parametrize` here instead of the for loop?

Reply:
I'm not sure how to parametrize over attributes of `self`.

Reply:
`offset = getattr(self, 'offset1')` etc

Reply:
Updated